src/battle-tooltips.ts

```diff
@@ -858,10 +858,10 @@ class BattleTooltips {
 					stats.spa = Math.floor(stats.spa * 1.5);
 				}
 				let allyActive = clientPokemon && clientPokemon.side.active;
-				if (allyActive && allyActive.length > 1) {
+				if (allyActive && allyActive.length > 0) {
```

Comment:
Checking whether `allyActive` is non-empty seems unnecessary when merely iterating over it (which for empty arrays does nothing).

Reply:
Yeah, fair point. The code in the condition used to look differently halfway through this and I never changed that back.